requirements.txt

```diff
@@ -1,4 +1,4 @@
 # These are Python requirements needed to run ceph-ansible main
-ansible-core>=2.12,<2.13
+ansible-core>=2.15,<2.16,!=2.9.10
```

Comment:
why keep !=2.9.10? with the previous conditions there is no chance to have that version installed

Reply:
it doesn't hurt, but yes, there's no need to keep it